src/cloudFunctionClient.ts

```diff
@@ -196,13 +198,29 @@ export class CloudFunctionClient {
   async deploy(cf: CloudFunction): Promise<cloudfunctions_v1.Schema$Operation> {
     const authClient = await this.getAuthClient();
     const deployedFunctions = await this.listFunctions();
-    const zipPath = await zipDir(cf.sourceDir);
+    const zipPath = path.join(
+      os.tmpdir(),
+      `cfsrc-${Math.floor(Math.random() * 100000)}.zip`,
+    );
+    await zipDir(cf.sourceDir, zipPath)
```

Comment:
Use `await` in a try/catch statement without then/catch or just use the then/catch.

Reply:
makes sense, done